dockstore-launcher/src/main/resources/plugins.json

```diff
@@ -5,6 +5,12 @@
     "name": "dockstore-file-s3-plugin",
     "version": "0.0.3"
   },{
+    "name": "dockstore-file-s3cmd-plugin",
```

Comment:
I would not add s3cmd as a default plug-in (default s3 handling many cases) and I would choose either icgc-get or icgc-storage depending on which one seems more reliable